v2/component_analyses.yaml

```diff
@@ -0,0 +1,188 @@
+openapi: 3.0.1
+info:
+  title: Analytics Core API Specs for Version 2
+  version: "2"
+servers:
+- url: https://f8a-analytics-2445582058137.production.gw.apicast.io/api/v2
+  description: "Dependency Analytics Server"
+security:
+- tokenAuth: []
+paths:
+  /component-analyses/{eco}/{name}/{ver}:
+    get:
+      tags:
+      - Scan Services
+      summary: Get security and other details about a software component and recommendations
+        to change to a more secured version
+      operationId: da_scanner.api_v2.component-analyses
+      parameters:
+      - name: eco
+        in: path
+        description: component ecosystem
+        required: true
+        schema:
+          type: string
+      - name: name
+        in: path
+        description: component name
+        required: true
+        schema:
+          type: string
+      - name: ver
+        in: path
+        description: component version
+        required: true
+        schema:
+          type: string
+      - name: uuid
+        in: header
+        description: User Identification uuid
+        required: false
```

Comment:
I would rather make is a required field and send some value as NA if the value is not available. This will help us understand clearly that the uuid is not present rather than think whether uuid is missing or maybe header is missing etc.

Reply:
I dont see the uuid in the response. am i missing something?

Reply:
Ack!